docs/streams/developer-guide.html

```diff
@@ -2795,7 +2795,7 @@
     <p>
         Introduced in 0.11.0 is a new broker config that is particularly relevant to Kafka Streams applications, <code>group.initial.rebalance.delay.ms</code>.
         This config specifies the time, in milliseconds, that the <code>GroupCoordinator</code> will delay the initial consumer rebalance.
-        The rebalance will be further delayed by the value of <code>group.initial.rebalance.delay.ms</code> as each new member joins the consumer group, up to a maximum of the value set by <code>max.poll.interval.ms</code>.
+        The re-balance will be further delayed by the value of <code>group.initial.rebalance.delay.ms</code> as each new member joins the consumer group, up to a maximum of the value set by <code>max.poll.interval.ms</code>.
```

Comment:
Nit: I think "rebalance" is fine as we are using this variant (rather than re-balance) all over the place -- including the names of configuration settings like `group.initial.rebalance.delay.ms`.

Reply:
yeah, agreed. i was using an overzealous sublime spellchecker.